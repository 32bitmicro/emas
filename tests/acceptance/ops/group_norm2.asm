AW r0, r0
AC r0, r0
SW r0, r0
CW r0, r0
OR r0, r0
OM r0, r0
NR r0, r0
NM r0, r0
ER r0, r0
EM r0, r0
XR r0, r0
XM r0, r0
CL r0, r0
LB r0, r0
RB r0, r0
CB r0, r0
