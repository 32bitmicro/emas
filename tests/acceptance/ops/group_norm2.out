@ 0x0000 : 0x8000  /  100 000 0 000 000 000  /  32768
@ 0x0001 : 0x8400  /  100 001 0 000 000 000  /  33792
@ 0x0002 : 0x8800  /  100 010 0 000 000 000  /  34816
@ 0x0003 : 0x8c00  /  100 011 0 000 000 000  /  35840
@ 0x0004 : 0x9000  /  100 100 0 000 000 000  /  36864
@ 0x0005 : 0x9400  /  100 101 0 000 000 000  /  37888
@ 0x0006 : 0x9800  /  100 110 0 000 000 000  /  38912
@ 0x0007 : 0x9c00  /  100 111 0 000 000 000  /  39936
@ 0x0008 : 0xa000  /  101 000 0 000 000 000  /  40960
@ 0x0009 : 0xa400  /  101 001 0 000 000 000  /  41984
@ 0x000a : 0xa800  /  101 010 0 000 000 000  /  43008
@ 0x000b : 0xac00  /  101 011 0 000 000 000  /  44032
@ 0x000c : 0xb000  /  101 100 0 000 000 000  /  45056
@ 0x000d : 0xb400  /  101 101 0 000 000 000  /  46080
@ 0x000e : 0xb800  /  101 110 0 000 000 000  /  47104
@ 0x000f : 0xbc00  /  101 111 0 000 000 000  /  48128
